test/behaviour/concept/thing/attribute/BUILD

```diff
@@ -18,34 +18,49 @@
 #
 
 package(default_visibility = ["//test/behaviour:__subpackages__"])
+
 load("@graknlabs_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
-load("@graknlabs_common//test/server:rules.bzl", "grakn_java_test")
+load("//test/behaviour:rules.bzl", "grakn_core_and_cluster_behaviour_test")
 
 java_library(
     name = "steps",
     srcs = [
-        "AttributeSteps.java"
+        "AttributeSteps.java",
     ],
+    visibility = ["//visibility:public"],
     deps = [
         # Internal Package Dependencies
         "//:client-java",
         "//test/behaviour/concept/thing:steps",
-        "//test/behaviour/util:util",
-        "//test/behaviour/connection:steps",
+        "//test/behaviour/util",
+        "//test/behaviour/connection:steps-base",
 
         # External Maven Dependencies
         "@maven//:junit_junit",
         "@maven//:io_cucumber_cucumber_java",
     ],
-    visibility = ["//visibility:public"],
 )
-
-grakn_java_test(
+grakn_core_and_cluster_behaviour_test(
```

Comment:
Couple of issues here I think:

Firstly the name doesn't have "java" in it anymore, despite us having Java, Python and NodeJS flavours of this macro.

Secondly the name is really freaking long and I don't feel like adding `core_and_cluster` contributes much. Together, we see Core and Cluster as Grakn. I think `grakn_java_cucumber_test` would be enough, or `grakn_java_bdd_test`, or even just leave it as `grakn_java_test`.

(Now I know it expands, currently, to _two_ `grakn_java_test` targets. So I reckon ideally we would rename _that_ to something else, like `grakn_java_bdd_test_internal` or `grakn_java_bdd_test_specific_server`)

What I do like is the use of a macro in the first place - that's a much better solution than just duplicating `test` everywhere 🙂 Now the hope is that developers are able to figure out that the targets are actually named `test-core` and `test-cluster` (or whatever we named them)

Reply:
Yes, I was hoping to exchange ideas with regards to the name. I've thought of other names:

- `grakn_bdd_java_test`
- `grakn_behaviour_java_test`

I'm thinking of keeping the suffix `java_test` since it's consistent with Bazel, and with our other rule `host_compatible_java_test`.

I'm leaning towards `grakn_behaviour_java_test`. What do you think?

Reply:
Yep I like `grakn_behaviour_java_test`.